data_tools/code_merge.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+import json
+
+from core_data_modules.data_models import Message, Label, Origin
+from core_data_modules.logging import Logger
+from core_data_modules.traced_data import Metadata
+from core_data_modules.util import TimeUtils
+
+Logger.set_project_name("CodeMerge")
+log = Logger(__name__)
+
+parser = argparse.ArgumentParser(description="Performs a code merge on a local dataset. "
+                                             "To use with Coda, use get.py, code_merge.py, then set.py")
+parser.add_argument("messages_input_file_path", metavar="messages-input-file-path",
+                    help="Path to the Coda messages file to perform the code merge on")
```

Comment:
```suggestion
                    help=" Path to the Coda messages file to read the messages -> to be merged codes from")
```

Reply:
Updated to "Path to the file to read the Coda messages to be code-merged from"